api/application.go

```diff
@@ -21,7 +21,8 @@ const (
 	ApplicationTagRoot   = ApplicationTagsRoot + "/:" + ID2
 	ApplicationFactsRoot = ApplicationRoot + "/facts"
 	ApplicationFactRoot  = ApplicationFactsRoot + "/:" + Key
-	AppBucketRoot        = ApplicationRoot + "/bucket/*" + Wildcard
+	AppBucketRoot        = ApplicationRoot + "/bucket"
+	AppBucketContentRoot = AppBucketRoot + "/*" + Wildcard
```

Comment:
**hint**: AppBucketRoot repurposed to be just /bucket to support bucket root browsing for troubleshooting.  The _BucketContent_ is for accessing bucket content.